test/distributed-ranges/shp/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+if(${CMAKE_VERSION} VERSION_LESS "3.26")
+    message(WARNING "disabled distributed ranges tests, use cmake>=3.26")
+    return()
+else ()
+    message(STATUS "suitable for distributed ranges cmake version ${CMAKE_VERSION} found")
+endif()
+
+find_package(fmt QUIET)
+if(fmt_FOUND)
+    if(fmt_VERSION VERSION_GREATER_EQUAL 8.1 AND fmt_VERSION VERSION_LESS 10)
+        message(STATUS "suitable for distributed ranges libfmt version ${fmt_VERSION} found")
+    else()
+        message(WARNING "libfmt version ${fmt_VERSION} is not >= 8.1 and < 10, skipping distributed ranges tests")
+        return()
+
+    endif()
+else()
+    message(WARNING "libfmt not found, install libfmt-dev 8.x or 9.x to enable distributed ranges tests")
+    return()
+endif()
+
+find_package(GTest)
+if (NOT GTest_FOUND)
+  message(WARNING "disabled distributed ranges tests, install libgtest-dev or other package with GTest")
+  return()
+endif ()
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_executable(
```

Comment:
Does it make sense for this file to up one level, since it is reaching into a sibling directory?

Reply:
It makes sense to keep it on this level because there will be another sibling "mhp" in the future and both shp and mhp CMakeLists.txt files should be in different directories.